### PR TITLE
[expo-video] Add `expo-video` docs page

### PR DESCRIPTION
**Pull request by behenate**
# Why

Expo-video is missing a docs page.

ENG-11349

# How

Renamed the `video.mdx` to `video-av.mdx` and assigned `video.mdx` as `expo-video` docs page. We should consider doing this the other way around (adding a suffix to `expo-video` file). Current way might create issues with linking to the docs, eg. if someone wants to redirect people to the latest `expo-av` video docs so they would link to `https://docs.expo.dev/versions/latest/sdk/video/`, but the users will be redirected to `expo-video` docs after SDK 51. I kept it this way in case we believe people should be redirected to `expo-video` anyways. Let me know what you think about it :)

Created a simple example for `expo-video` (similar to the `expo-av` example), which showcases how to use player properties and events. 

# Test Plan

Tested by running the docs locally

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/video-av.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+packageName: 'expo-av'
+iconUrl: '/static/images/packages/expo-av.png'
+platforms: ['android', 'ios', 'web']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { SnackInline } from '~/ui/components/Snippet';
+
+The `Video` component from **`expo-av`** displays a video inline with the other UI elements in your app.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/video-av.mdx:14]:
```suggestion
> **info** The `Video` component from expo-av, which is documented on this page, will be replaced by an improved version in expo-video in an upcoming release (when the new library is stable). [Learn about expo-video](video.mdx)

The `Video` component from **`expo-av`** displays a video inline with the other UI elements in your app.
```

File: docs/pages/versions/unversioned/sdk/video.mdx [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 ---
 
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import { SnackInline } from '~/ui/components/Snippet';
 
-The `Video` component from **`expo-av`** displays a video inline with the other UI elements in your app.
-
-Much of Video and Audio have common APIs that are documented in [AV documentation](av.mdx). This page covers video-specific props and APIs. We encourage you to skim through this document to get basic video working, and then move on to [AV documentation](av.mdx) for more advanced functionality. The audio experience of video (such as whether to interrupt music already playing in another app, or whether to play sound while the phone is on silent mode) can be customized using the [Audio API](audio.mdx).
+A cross-platform, performant video component for React Native and Expo with Web support.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/video.mdx:13]:
```suggestion
> **info** expo-video is a new, experimental package that aims to replace the `Video` component from expo-av with a more modern and reliable implementation. If you are concerned about using a new package, consider [expo-av](video.mdx) until expo-video has stabilized.

A cross-platform, performant video component for React Native and Expo with Web support.
```

File: docs/pages/versions/unversioned/sdk/video.mdx [hunk 3]
```diff
@@ -1,19 +1,16 @@
 ---
-title: Video
-description: A library that provides an API to implement video playback and recording in apps.
-sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-av'
-packageName: 'expo-av'
-iconUrl: '/static/images/packages/expo-av.png'
+title: Video (expo-video)
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/video.mdx:2]:
```suggestion
title: Video
```

**Review by brentvatne — APPROVED** [review]

**Review by Simek — APPROVED** [review]
Thanks for the updates! 👍  

As discussed offline, we will be working further to improve those docs in near future, but  it's definitely a great start!

File: packages/expo-video/src/VideoView.tsx [hunk 4]
```diff
@@ -13,6 +13,11 @@ import NativeVideoModule from './NativeVideoModule';
 import NativeVideoView from './NativeVideoView';
 import { VideoPlayer, VideoSource, VideoViewProps } from './VideoView.types';
 
+/**
+ * Creates a `VideoPlayer`, which will be automatically cleaned up when the component is unmounted.
+ * @param source - A video source to initialize the player with
```
Comment (amandeepmittal) [inline packages/expo-video/src/VideoView.tsx:18]:
```suggestion
 * @param source - A video source that is used to initialize the player.
```

Let's rephrase this to make it more clear and also add the missing period.

**Review by amandeepmittal — APPROVED** [review]
Thank you adding this doc and for incorporating those changes
